node/bft-consensus/tests/common/state.rs

```diff
@@ -141,8 +141,17 @@ impl TestBftExecutionState {
 #[async_trait]
 impl ExecutionState for TestBftExecutionState {
     async fn handle_consensus_output(&self, consensus_output: ConsensusOutput) {
+        // Register and log some useful information.
+        let mut leader = consensus_output.sub_dag.leader.header.author.to_string();
+        leader.truncate(8);
```

Comment:
magic num

Reply:
it's not magic, it's just shortened for viewing convenience :smile: